CONTRIBUTING.md

```diff
@@ -230,6 +230,35 @@ jlpm run clean
 jlpm run build:packages
 ```
 
+## [Writing Documentation](#writing-documenation)
+
+Documentation is written in Markdown and reStructuredText.  In particular, the documentation on our Sphinx page is written in reStructuredText. To ensure that the Sphinx page builds, you'll need to install the documentation dependencies with `conda`.  These dependencies are located in `docs/enviornment.yml`
```

Comment:
I think we should call it our `Read the Docs` page, since that is where we host it (rather than Sphinx, the build tool)

Reply:
Typo in `environment`